jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -107,11 +107,11 @@ public JibExtension(Project project) {
     extraDirectory.set(resolveDefaultExtraDirectory(project.getProjectDir().toPath()));
   }
 
-  public void from(Action<? super ImageParameters> action) {
+  public void from(Action<? super BaseImageParameters> action) {
     action.execute(from);
   }
 
-  public void to(Action<? super ImageParameters> action) {
+  public void to(Action<? super TargetImageParameters> action) {
```

Comment:
It's possible we don't need `ImageParameters` anymore. It passes through some annotations, but we can probably apply those on the subclasses directly.

Reply:
Yeah, I think we should get rid of it.